src/utils/react.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect, useRef } from 'react'
+
+export let useStateLens = value => {
+  let [state, setState] = useState(value)
+  return { get: () => state, set: setState }
+}
+
+// https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+export let useInterval = (callback, delay = 1000) => {
```

Comment:
This is weird and only used for one story, so probably shouldn't be part of the lib's utils